docs/user-guide/grpc.md

```diff
@@ -110,9 +120,11 @@ serverOption := grpc.Creds(creds)
 var s *grpc.Server = grpc.NewServer(serverOption)
 defer s.Stop()
 
-helloworld.RegisterGreeterServer(s, &myserver{})
+pb.RegisterGreeterServer(s, &server{})
 err := s.Serve(lis)
 
+// s := grpc.NewServer()
```

Comment:
could you remove this line?

Reply:
we have to comment original code from grpc-go

Reply:
please remove